atlasdb-workload-server-api/src/main/java/com/palantir/atlasdb/workload/transaction/TransactionAction.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.workload.transaction;
+
+import com.palantir.atlasdb.workload.store.WorkloadCell;
+import org.immutables.value.Value;
+
+public interface TransactionAction {
+
+    /** Cell (Key, Column) to apply the action to. */
+    @Value.Parameter
+    WorkloadCell cell();
```

Comment:
Maybe we should introduce some notion of cell-level transaction actions - in the hopefully not-too-distant future we'll want to support getRange etc. that aren't cell-based